exports.go

```diff
@@ -124,6 +126,28 @@ func (e *Export) Validate(vr *ValidationResults) {
 		e.Latency.Validate(vr)
 	}
 	e.Subject.Validate(vr)
+
+	if e.AccountTokenPosition > 0 {
+		if !e.Subject.HasWildCards() {
+			vr.AddError("Account Token Position can only be used when wildcard subjects: %s", e.Subject)
+		} else {
+			subj := string(e.Subject)
+			token := strings.Split(subj, ".")
+			tkCnt := uint(len(token))
+			if e.AccountTokenPosition == tkCnt && strings.HasSuffix(subj, "*") {
```

Comment:
I think that the account token position needs to be an independent token and it needs to match `*`. That is if your `account_token_position` is set to `1` and you have a subject like `foo.>`, that should fail validation. The subject in question should be perhaps `foo.*.>` or `foo.*.*`, but at any rate, the intent of what token represents the account should be explicit on the subject template as well.

Reply:
so you are saying the token position needs to match a * and can't match a token covered by > ?

Reply:
@aricart , made it function with * only